### PR TITLE
updating dependencies

### PR DESCRIPTION
**Pull request by chrisnojima**
Update our babel and electron (and some small ones) and add instructions for how to do this in the future

@keybase/react-hackers

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: desktop/README.md [hunk 1]
```diff
@@ -61,3 +61,16 @@ If you have a crash of a non-sourcemapped build (like a screenshot with a crash)
 npm install -g sourcemap-finder
 smfinder --position 1200:10 path-to-your-source-map
 ```
+
+### Dependencies
+
+Updating packages
+
+Some of these steps are theoretically not needed but in practice npm is buggy as hell. Make sure you're npm is updated globally
```
Comment (cjb) [inline desktop/README.md]: your

File: desktop/README.md [hunk 2]
```diff
@@ -61,3 +61,16 @@ If you have a crash of a non-sourcemapped build (like a screenshot with a crash)
...
+```npm install -g npm```
+
+1. Remove old shrinkwrap: Run ```rm  npm-shrinkwrap.json```
```
Comment (cjb) [inline desktop/README.md]: extra space between rm  npm

**Comment by cjb** [comment]
Tested, works :thumbsup:

**Pull request merged**